smoke-tests/src/test/java/io/opentelemetry/smoketest/TestContainerManager.java

```diff
@@ -28,4 +28,9 @@
       String[] command);
 
   void stopTarget();
+
+  static boolean useLinuxContainers() {
```

Comment:
This method's result is always reverted, should we revert the method itself instead?

Reply:
makes sense 👍